UnitsNet/GeneratedCode/Quantities/ElectricChargeDensity.g.cs

```diff
@@ -642,24 +642,44 @@ public ElectricChargeDensity ToUnit(ElectricChargeDensityUnit unit)
         public ElectricChargeDensity ToUnit(ElectricChargeDensityUnit unit, UnitConverter unitConverter)
         {
             if(Unit == unit)
+            {
+                // Already in requested units.
                 return this;
+            }
+            else if(unitConverter.TryGetConversionFunction((typeof(ElectricChargeDensity), Unit, typeof(ElectricChargeDensity), unit), out var conversionFunction))
+            {
+                // Direct conversion to requested unit found. Return the converted quantity.
+                var converted = conversionFunction(this);
+                return (ElectricChargeDensity)converted;
+            }
+            else if(Unit != BaseUnit)
```

Comment:
I know the code base is riddled with varying styles, and this is nitpick, but I notice `if()` without whitespace is consistently used instead of `if ()`. 

I believe the latter is the most common style in C#, and used in [Microsoft's C# coding style conventions](https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#layout-conventions).

Can we use the latter form?
Then we can clean the old code to be consistent.

Reply:
These blocks don't have a space between ```if (``` ?

Reply:
I'll fix it post merge in a solution wide fix, but what I meant is that I would like syntax `if (xxx)` instead of `if(xxx)`.